hooks/playbooks/disable_catalogsource.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Disable/Enable default CatalogSource
+  hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Get all the openshift marketplace pods
+      environment:
+        KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+        PATH: "{{ cifmw_path }}"
+      ansible.builtin.command:
+        cmd: >-
+          oc get pods -o name -n openshift-marketplace
+      register: _pod_list
+
+    - name: Get the redhat-operators pods having CrashLoopBackOff state
+      kubernetes.core.k8s_info:
+        kind: Pod
+        kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+        name: "{{ pod_list.stdout | regex_search('^pod/redhat-operators-.*$', multiline=True) | split('/') | last }}"
```

Comment:
Uhmm I don't get this one, this will query a single pods as your are passing the name field. Isn't enough if you remove the previous task and the name? The field_selector here should do the rest.

Reply:
Yes, got it now! Updated it.